### PR TITLE
Inverse Interface Switch for snmp-interface

### PR DESCRIPTION
**Pull request by tweippert**
Implemented -i switch to snmp-interface command configuration. As for Interface Monitoring it is sometime needed to inverse the status (down-> OK, up -> NOT OK).

Can be configured via snmp_interface_inverse variable (boolean). Add documentation, too.

### CODE REVIEW

File: itl/command-plugins-manubulon.conf [hunk 1]
```diff
@@ -183,6 +183,10 @@ object CheckCommand "snmp-interface" {
 		"-e" = {
 			set_if = "$snmp_interface_errors$"
 		}
+		"-i" = {
+			set_if = "$snmp_interface_inverse$"
+		}
+		"-r" = {
```
Comment (h3o66) [inline itl/command-plugins-manubulon.conf]: Seems like you added the switch (-r) a second time in the file. Before someone can merge this you should fix this

Comment (tweippert) [inline itl/command-plugins-manubulon.conf]: Yes, stupid copy & paste error ... fixed in new commit.

Comment (h3o66) [inline itl/command-plugins-manubulon.conf]: Thanks :+1:

**Comment by dnsmichi** [comment]
Thanks for the patch @tweippert and thanks for the kind review @h3o66 👍 Fetch and squashed for pushing this upstream.

**Pull request closed without merging**